static/app/utils/profiling/renderers/cursorRenderer.tsx

```diff
@@ -0,0 +1,39 @@
+import {mat3, vec2} from 'gl-matrix';
+
+import {FlamegraphTheme} from '../flamegraph/FlamegraphTheme';
+import {getContext, Rect, resizeCanvasToDisplaySize} from '../gl/utils';
+
+class CursorRenderer {
+  canvas: HTMLCanvasElement;
+  context: CanvasRenderingContext2D;
+
+  theme: FlamegraphTheme;
+
+  constructor(canvas: HTMLCanvasElement, theme: FlamegraphTheme) {
+    this.canvas = canvas;
+    this.theme = theme;
+    this.context = getContext(canvas, '2d');
+
+    resizeCanvasToDisplaySize(canvas);
+  }
+
+  draw(configSpaceCursor: vec2, physicalSpace: Rect, configToPhysicalSpace: mat3): void {
+    const physicalSpaceCursor = vec2.transformMat3(
+      vec2.create(),
+      vec2.fromValues(configSpaceCursor[0], configSpaceCursor[1]),
+      configToPhysicalSpace
+    );
```

Comment:
Is there a reason you need a copy of `configSpaceCursor` here? If I'm understanding this correctly, you can directly use `configSpaceCursor` here as `vec2.transformMat3` doesn't mutate the second argument, and you don't need the copy anywhere else.

```suggestion
    const physicalSpaceCursor = vec2.transformMat3(
      vec2.create(),
     configSpaceCursor,
      configToPhysicalSpace
    );
```

Reply:
Yup, correct, probably an artifact from older days.